### PR TITLE
Video call

### LINKED ISSUE #108: Implement video call

### PR DESCRIPTION
**Pull request by AttiaYoussef**
Resolves: #108 

# Added

- Join video room
- Share local video
- Get remote video
- Choose which video to focus on
- Add possibility to switch cameras

# TO DO

- [x] Fix tests
- [x] Have a better coverage

### CODE REVIEW

**Review by MRandl — APPROVED** [review]
great work as far as i have seen

File: app/src/main/java/io/github/polysmee/agora/VoiceCall.java [hunk 1]
```diff
@@ -88,46 +79,46 @@ public int leaveChannel() {
...
      * @return the result of the mute method of RtcEngine (0 if success, not 0 otherwise)
      */
-    public int mute(boolean mute) {
+    /*public int mute(boolean mute) {
         int result = mRtcEngine.muteLocalAudioStream(mute);
         appointment.muteUser(new DatabaseUser(MainUserSingleton.getInstance().getId()), mute);
 
         return result;
-    }
+    }*/
```
Comment (MrZaiko) [inline app/src/main/java/io/github/polysmee/agora/VoiceCall.java:95]: Maybe remove this file since everything is commented out.

**Review by MrZaiko — APPROVED** [review]
I saw a lot of .png files. At some point, we'll probably need to change to svg. This is clearly not a problem right now.

Very good work !

File: app/src/main/java/io/github/polysmee/agora/VoiceCall.java (hunk 1 above)
Comment (AttiaYoussef) [reply app/src/main/java/io/github/polysmee/agora/VoiceCall.java:95]: I'll do that as soon as I explain to Adrien everything I changed so he's not lost.

**Pull request merged**